### DISCUSSION TITLE
Wiki page missing

### DISCUSSION THREAD

**Discussion by jsfan3** [searx/searx#3175] [Installation issue: Official wiki]
Why is the following page disappeared?
https://github.com/asciimoo/searx/wiki/How-to-create-and-configure-SSL

I found it in the Web Archive:
https://web.archive.org/web/20190620154821/https://github.com/asciimoo/searx/wiki/How-to-create-and-configure-SSL

IMHO, this page is very helpful, so I suggest to keep it available.

**Comment by unixfox** [comment]
Because the wiki pages moved to https://searx.github.io/searx/

**Reply by jsfan3** [reply]
The wiki pages at the address https://searx.github.io/searx/ don't contain any instructions to enable HTTPS.
That's why I needed the page archived on https://web.archive.org/web/20190620154821/https://github.com/asciimoo/searx/wiki/How-to-create-and-configure-SSL.

**Reply by unixfox** [reply]
> The wiki pages at the address https://searx.github.io/searx/ don't contain any instructions to enable HTTPS.
> That's why I needed the page archived on https://web.archive.org/web/20190620154821/https://github.com/asciimoo/searx/wiki/How-to-create-and-configure-SSL.

Probably because this tutorial is not recommended to be followed.

In the tutorial you will be generating a self signed certificate which your browser doesn't trust. If you follow the tutorial you will have a warning page when viewing the page of your searx installation.

Nowadays on apache just follow any tutorial about certbot on the internet and you will have a perfect HTTPS installation.

**Reply by jsfan3** [reply]
That's precisely what I did: I installed Searx via Docker and then configured Apache as a reverse proxy over http. Then, via Let's Encrypt, I added support for https. If I hadn't read this guide, though, it would never have occurred to me to use Apache as a reverse proxy. In that sense, it was helpful to me. 

My opinion is that a simple indication like this should be made explicit without assuming that we are all experts. I, e.g., before using searx, didn't even know what Docker was.

**Reply by unixfox** [reply]
Mmh ok but to be fair we recommend apache at the end of the normal install without docker: https://searx.github.io/searx/admin/installation.html#installation-scripts

And if you want an all in one package where everything is included: searx, reverse proxy and more then you are better off using this program: https://github.com/searx/searx-docker
